template/Goerli-DssSpell.sol

```diff
@@ -14,16 +14,12 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
```

Comment:
template file name could also be updated I guess

Reply:
Fixed!